--- miniaudio/setup.py	2025-09-21 13:58:00.549944952 -0700
+++ miniaudio.mod/setup.py	2025-09-21 14:00:24.565292324 -0700
@@ -6,7 +6,7 @@
 from setuptools import setup
 
 
-miniaudio_path = os.path.abspath(".")  # to make sure the compiler can find the required include files
+miniaudio_path = os.path.abspath(os.path.join(__file__, ".."))  # to make sure the compiler can find the required include files
 PKG_VERSION = re.search(r'^__version__\s*=\s*"(.+)"', open("miniaudio.py", "rt").read(), re.MULTILINE).groups()[0]
 
 
@@ -27,8 +27,6 @@
         py_modules=["miniaudio"],
         install_requires=["cffi>=1.12.0"],
         # setup_requires=["cffi>=1.12.0", "wheel"],
-        tests_require=[],
-        test_suite="setup.miniaudio_test_suite"
     )
 
 
